src/applications/gi/sass/partials/_gi-search-page.scss

```diff
@@ -159,6 +161,25 @@
         margin-right: 0.4em;
       }
     }
+  }
+
+  .vettec-logo-container {
+    margin-top: -32px;
+    margin-bottom: 16px;
```

Comment:
For consistency, can we express the margins and paddings in here in `em` instead of `px`?

Reply:
Tried to switch to em and this caused the image to be skewed incorrectly

Reply:
done